zig/neotest_runner.zig

```diff
@@ -1,6 +1,14 @@
 const std = @import("std");
 const builtin = @import("builtin");
 
+const platforrm_stub = if (builtin.os.tag == .windows) struct {
```

Comment:
This could be simplified into:
```diff
diff --git a/zig/neotest_runner.zig b/zig/neotest_runner.zig
index ff365d9..0a1ecc1 100644
--- a/zig/neotest_runner.zig
+++ b/zig/neotest_runner.zig
@@ -1,13 +1,10 @@
 const std = @import("std");
 const builtin = @import("builtin");
 
-const platforrm_stub = if (builtin.os.tag == .windows) struct {
-    const platform = @import("platform/windows/platform.zig");
-} else struct {
-    const platform = @import("platform/posix/platform.zig");
-};
-
-const platform = platforrm_stub.platform;
+const platform = if (builtin.os.tag == .windows)
+    @import("platform/windows/platform.zig")
+else
+    @import("platform/posix/platform.zig");
 
 pub const std_options: std.Options = .{
     .log_level = .debug,
```

Reply:
Thanks for catching that. I tried that originally and was getting errors. I think the errors may have been unrelated at the time :)

I fixed the simplification :)